modules/vpn_ha/main.tf

```diff
@@ -106,6 +106,9 @@ resource "google_compute_router_peer" "bgp_peer" {
   router          = local.router
   peer_ip_address = each.value.bgp_peer.address
   peer_asn        = each.value.bgp_peer.asn
+  ip_address = (
```

Comment:
@imrannayer 
It may not be necessary, but they are attached because the line would be very long in that case.
Same as advertise_mode in bgp_peer_options.

Reply:
@tnk-ysk If it is not needed then plz remove it and keep it in one line.